pkg/apiserver/authentication/authenticators/bearertoken/bearertoken.go

```diff
@@ -16,14 +16,18 @@ func New() authenticator.Token {
 	return &tokenAuthenticator{}
 }
 
-func (a *tokenAuthenticator) AuthenticateToken(ctx context.Context, token string) (*authenticator.Response, bool, error) {
+func (a *tokenAuthenticator) AuthenticateToken(ctx context.Context, token string) (response *authenticator.Response, ok bool, err error) {
 	issuer := jwt.NewTokenIssuer("", time.Second)
```

Comment:
IMO, the JWT secret should be set correctly.

Reply:
The documentation of `tokenAuthenticator` type should be rectified as well.
```
// tokenAuthenticator implements an anonymous auth
type tokenAuthenticator struct{}
```

Reply:
> IMO, the JWT secret should be set correctly.

It's not in this case. As you can see, I use `VerifyWithoutClaimsValidation`. So, we don't need a real token here.

Reply:
> The documentation of `tokenAuthenticator` type should be rectified as well.
> 
> ```
> // tokenAuthenticator implements an anonymous auth
> type tokenAuthenticator struct{}
> ```

Sure.

Reply:
Alright

Reply:
Sure, please check https://github.com/kubesphere-sigs/ks-devops/pull/56/commits/8e702d811c2bcde6ac197638d5322dfbf370fb55